### PR TITLE
[perl/de] translate perl to german

### PR DESCRIPTION
**Pull request by Git-Jiro**
First version of a translation of the Perl page to german. Please review for typos and spelling mistakes.

### CODE REVIEW

File: de-de/perl-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+name: perl
+category: language
```
Comment (vendethiel) [inline de-de/perl-de.html.markdown]: please remove this, then I'll merge it. Sorry for the delay.

Comment (Git-Jiro) [inline de-de/perl-de.html.markdown]: I hope I removed the correct lines now.

File: de-de/perl-de.html.markdown [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+contributors:
+    - ["Korjavin Ivan", "http://github.com/korjavin"]
+translators:
+    - ["Martin Schimandl", "http://github.com/Git-Jiro"]
+lang: de-de
+---
+
+Perl 5 ist eine sehr mächtige, funktionsreiche Programmiersprache mit über 25 Jahren Entwicklungsgeschichte.
+
+Perl 5 läuft auf über 100 Platformen von portablen Geräten bis hin zu Mainframes. Perl 5 ist geeignet für Rapid-Prototyping und auch groß angelegte Entwicklungs-Projekte.
```
Comment (geoffliu) [inline de-de/perl-de.html.markdown]: portabelen

File: de-de/perl-de.html.markdown [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+sub logger {
+  my $logmessage = shift;
+
+  open my $logfile, ">>", "my.log" or die "Could not open my.log: $!";
+
+  print $logfile $logmessage;
+}
+
+# Nun könnne wir die Subroutine genau wie eine eingebaute Funktion verwenden:
```
Comment (geoffliu) [inline de-de/perl-de.html.markdown]: können

File: de-de/perl-de.html.markdown [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+logger("We have a logger subroutine!");
+```
+
+#### Verwenden von Perl Modulen
+
+Perl Module lieferen eine Menge an Funktionen die dabei Helfen das Rad nicht neu erfinden zu müssen. Perl Module können von CPAN (http://www.cpan.org/) heruntergeladen werden. Einige populäre Module sind in der Perl Distribution selbst bereits enthalten.
```
Comment (geoffliu) [inline de-de/perl-de.html.markdown]: liefern

File: de-de/perl-de.html.markdown [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+if (/foo/)       { ... }  # Wahr wenn "foo" in $_ enthalten ist
+if ($a =~ /foo/) { ... }  # Wahr wenn "foo" in $a enthalten ist
+
+# Einfache Substitution
+
+$a =~ s/foo/bar/;         # Ersetzt foo mit bar in $a
+$a =~ s/foo/bar/g;        # Ersetzt ALLE VORKOMMNISSE von foo mit bar in $a
+
+
+#### Datien und Ein-/Ausgabe
```
Comment (geoffliu) [inline de-de/perl-de.html.markdown]: Dateien

File: de-de/perl-de.html.markdown [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+Perl 5 ist eine sehr mächtige, funktionsreiche Programmiersprache mit über 25 Jahren Entwicklungsgeschichte.
+
+Perl 5 läuft auf über 100 Platformen von portablen Geräten bis hin zu Mainframes. Perl 5 ist geeignet für Rapid-Prototyping und auch groß angelegte Entwicklungs-Projekte.
+
+```perl
+# Einzeilige Kommentare beginnen mit dem # Symbol.
+
+
+#### Perl variablen typen
```
Comment (geoffliu) [inline de-de/perl-de.html.markdown]: Capitalize

File: de-de/perl-de.html.markdown [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+  print;
+}
+
+# Die Perl-Eigene Post-Bedingungs-Schreibweise nochmals
+print for @elements;
+
+#### Reguläre Ausdrücke
+
+# Die Unterstützung von Perl für reguläre Ausdrücke ist weit und tiefgreifend.
+# Sie ist ausführlichst in perlrequick, perlretut und sonstwo dokumentiert.
```
Comment (geoffliu) [inline de-de/perl-de.html.markdown]: "ausführlichst"?

**Comment by geoffliu** [comment]
Sorry for the delay @Git-Jiro, we're catching up to the old PRs.

**Comment by Git-Jiro** [comment]
Hi @geoffliu, no problem! ;-)

**Comment by geoffliu** [comment]
Danke schön :)

**Pull request merged**